### DISCUSSION TITLE
Inverted axis only in program

### DISCUSSION THREAD

**Discussion by smitroast** [terjeio/ioSender#304] [General]
Hi all had everything on my cnc running perfect, until I turned it on today I have checked and tried everything I have looked at every grbl setting in ioSender but nothing helps.
 
 jogging sends the machine in the right direction for each axis,
Cycle start and it goes the wrong way….
I have changed origin in fs360 inverted the axis and inverts the whole program not the axis although it runs it’s upside down. I had it homing will limit switches last night now it does a 10mm jig while homing and come sup with an alarm any help would be appreciated

I’m using a makerstore E5XMCS1.4 thank you

if you haven’t heard of it

“E5X- 5 axis plus cyclone”
“MCS- multi control system”
“1.4 - teensy 1.4”

**Comment by terjeio** [comment]
Odd, what about motion commands sent from the MDI. They are going the wrong way as well?

**Comment by smitroast** [comment]
found out it was something to do with a whole heap of input inversion,etc
not sure what i did top fix it but i did hahah

On Sun, Jun 25, 2023 at 5:12 PM Terje Io ***@***.***> wrote:

> Odd, what about motion commands sent from the MDI. They are going the
> wrong way as well?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/terjeio/ioSender/discussions/304#discussioncomment-6272506>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A7LRIIS5OAWL6LYCNJZBZALXM7QHBANCNFSM6AAAAAAZSLOOI4>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

INSTAGRAM: @JBSMETALWORKS
FACEBOOK: @JBSMITHMETALWORKS
